scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -773,6 +801,91 @@ def log_event(self, event):
         finally:
             self._bookmark_lock.release()
 
+    def _param_placeholder_value(self, channel, provider, param):
+        if not re.match("^%%[0-9]+$", param):
```

Comment:
I guess I may miss something here, but it seems that every branch that starts from [there](https://github.com/scalyr/scalyr-agent-2/blob/d0d45d49327087a454d8be6900e4307dcd82a925/scalyr_agent/builtin_monitors/windows_event_log_monitor.py#L867)  already does the `^%%[0-9]+$` match and only then calls the `_param_placeholder_value ` function.  Does this mean that this check will always and up with false?

Reply:
You're right, I am just being cautious here because _param_placeholder_value expects that format for param (later it does `int(param[2:])`).  This extra check is to quickly identify a problem if something else gets passed into the function in a future change.

Reply:
Yeah, got it.